rel/i18n/emqx_limiter_schema.hocon

```diff
@@ -1,9 +1,9 @@
 emqx_limiter_schema {
 
 max_conn_rate.desc:
-"""Used to limit the rate at which the current listener accepts connections.
+"""Limits how quickly this listener accepts connections, per each node.
 
-Once the limit is reached, EMQX will pause fetching connections from the Accept queue, thereby delaying or rejecting new connections.
+Once the limit is reached, EMQX will pause serving connections from the Accept queue, thereby delaying or rejecting new connections.
```

Comment:
this is no longer the case since 5.3.0
see
https://github.com/emqx/emqx/pull/14219
and
https://github.com/emqx/esockd/pull/196